### DISCUSSION TITLE
Is it possible to add other accounts/profiles to lists with Tusky?

### DISCUSSION THREAD

**Discussion by pauljacobson** [tuskyapp/Tusky#2958] [Q&A]
Hi there

I started using Tusky v20.0 today and I really like the app. I thought that it would be possible to add profiles to lists but I can't seem to find the option. I'm looking here when I view a profile:

![Screenshot_20221201_165620](https://user-images.githubusercontent.com/2912202/205085301-fc5453cf-1bd1-431e-ad9c-ea72ad39f617.png)

As a comparison, Fedilab has an option for this here:

![Screenshot_20221201-165754_Fedilab](https://user-images.githubusercontent.com/2912202/205085851-9ebfcc5a-b913-4c80-b878-4c5d70a25939.png)

Am I just missing the feature in Tusky?

**Comment by Tak** [comment]
You can currently edit lists from the Lists entry in the main menu, and there's a [work in progress](https://github.com/tuskyapp/Tusky/pull/2818) to enable it from the profile view

**Reply by pauljacobson** [reply]
Great, thank you
